src/doc/src/reference/specifying-dependencies.md

```diff
@@ -315,10 +315,10 @@ syntax](../../reference/conditional-compilation.html) will be used to define
 these sections:
 
 ```toml
-[target.'cfg(windows)'.dependencies]
+[target.'cfg(target_os = "windows")'.dependencies]
```

Comment:
Hmm, rustc seems to support `cfg(windows)` and `cfg(unix)`

See https://doc.rust-lang.org/reference/conditional-compilation.html#unix-and-windows

This likely needs a closer look